scripts/shared/nginx/main.conf

```diff
@@ -2,4 +2,9 @@ server {
   listen 80;
   listen [::]:80;
   include /etc/nginx/site-path-enabled/*.conf;
+  
+  root /var/www/files;
+  location / {
```

Comment:
I would not auto index
and if you don't auto index you don't need a /location

Reply:
Auto-indexing allows folders to be served by nginx without error. It will instead provide a simple file browser, showing everything in that directory. This is so that you can add a folder to the /var/www/files intuitively, without having to provide index.html file for each one so that accessing the folder directly doesn't give an error.

Reply:
Nvm, I've removed it now.